packages/api-gateway/src/drivers/http/routes/places/index.js

```diff
@@ -1,8 +1,8 @@
 const { placesAdapters } = require('../../adapters');
-const schema = require('./schema');
+const { postSchema } = require('./schema');
 
 async function placesRoute(fastify) {
-  await fastify.post('/', { schema }, placesAdapters.places);
+  await fastify.post('/create', { schema: postSchema }, placesAdapters.postPlace);
```

Comment:
@Alejandrocuartas as we are REST we don't need to put the `create` statement as part of the URL because the HTTP method indicates directly which kind of operation are we performing that in this case is POST that represents the creation of a resource.